packages/open-next/src/adapters/dynamo-provider.ts

```diff
@@ -31,45 +36,56 @@ export async function handler(
   }
 }
 
+/**
+ * Sending to dynamo X commands at a time, using about X * 25 write units per batch to not overwhelm DDB
+ * and give production plenty of room to work with. With DDB Response times, you can expect about 10 batches per second.
+ */
+const DYNAMO_BATCH_WRITE_COMMAND_CONCURRENCY = 4;
```

Comment:
Is there a reason why it's hardcoded to 4 ?
We could add an env variable to modify this behaviour, but keep it at 4 by default.